tests/unit/utils/rtf-to-html-conversion-test.ts

```diff
@@ -1,32 +1,66 @@
+    const expectedHtml = '<span>Lorem Ipsum</span>';
+    const htmlContent = oneLineTrim`
+    <html xmlns:o="urn:schemas-microsoft-com:office:office"
+        xmlns:w="urn:schemas-microsoft-com:office:word"
+        xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
+        xmlns="http://www.w3.org/TR/REC-html40">
 
-    const content = pasteHandler(rtfContent, htmlContent);
-    // when handling the RTF content, and the process is done successfully the pasteHandler -> cleanDocx function attaches <!--StartFragment--> in the output
-    assert.true(content.includes('<!--StartFragment-->'));
-  });
+        <head>
+        <meta http-equiv=Content-Type content="text/html; charset=utf-8">
+        <meta name=ProgId content=Word.Document>
+        <meta name=Generator content="Microsoft Word 15">
+        <meta name=Originator content="Microsoft Word 15">
+        </head>
+        <body lang=en-AL style='tab-interval:36.0pt;word-wrap:break-word'>
+        <!--StartFragment--><span style='font-size:14.0pt;font-family:"Calibri",sans-serif;
+        mso-ascii-theme-font:minor-latin;mso-fareast-font-family:Calibri;mso-fareast-theme-font:
+        minor-latin;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman";
+        mso-bidi-theme-font:minor-bidi;mso-ansi-language:#0C00;mso-fareast-language:
+        EN-US;mso-bidi-language:AR-SA'>Lorem Ipsum</span><!--EndFragment-->
+        </body>
+    </html>
+    `;
 
-  test('It should display clean HTML', function (assert) {
-    const rtfContent = '';
-    const htmlContent =
-      '<html xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:w="urn:schemas-microsoft-com:office:word" xmlns:m="http://schemas.microsoft.com/office/2004/12/omml" xmlns="http://www.w3.org/TR/REC-html40"> <head> <meta http-equiv=Content-Type content="text/html; charset=utf-8"> <meta name=ProgId content=Word.Document> <meta name=Generator content="Microsoft Word 15"> <meta name=Originator content="Microsoft Word 15"> </head> <body lang=en-AL> <!--StartFragment--><span style=\'font-size:14.0pt;font-family:"Calibri",sans-serif; mso-ascii-theme-font:minor-latin;mso-fareast-font-family:Calibri;mso-fareast-theme-font: minor-latin;mso-hansi-theme-font:minor-latin;mso-bidi-font-family:"Times New Roman"; mso-bidi-theme-font:minor-bidi;mso-ansi-language:#0C00;mso-fareast-language: EN-US;mso-bidi-language:AR-SA\'>of</span><!--EndFragment--> </body> </html>';
+    const inputParser = new HTMLInputParser({});
+    const actualHtml = convertMsWordHtml(rtfContent, htmlContent, inputParser);
 
-    const content = pasteHandler(rtfContent, htmlContent);
-    assert.strictEqual(content, htmlContent);
+    assert.strictEqual(expectedHtml, actualHtml);
   });
 
-  test('It should display "No content found!"', function (assert) {
-    const rtfContent = '';
-    const htmlContent = '';
+  test('sbx6 It should display clean HTML', function (assert) {
```

Comment:
what is sbx6? just curious :)

Reply:
ahah it's like a "nickname" my friends used to call me :D

Reply:
aha ok :)
you don't have to take it out, but it's also not necessary to add it in the future
we have `git blame` if we wanna see who wrote which piece of code